mirage/seed_image/catalog_seed_image.py

```diff
@@ -122,15 +123,12 @@ def make_seed(self):
                                                           self.coord_adjust['x']])).astype(np.int)
 
         # calculate the exposure time of a single frame, based on the size of the subarray
-        #self.calcFrameTime()
-        self.frametime = utils.calc_frame_time(self.params['Inst']['instrument'], self.params['Readout']['array_name'],
-                                         self.nominal_dims[0], self.nominal_dims[1], self.params['Readout']['namp'])
-        print("Frametime is {}".format(self.frametime))
+        self.calcFrameTime()
 
         # Read in the pixel area map, which will be needed for certain
         # sources in the seed image
         self.prepare_PAM()
-
+        
```

Comment:
This PR has a lot of trailing whitespace changes, which I expect are probably accidental. Perhaps some cleanup commit in master that hasn't been merged into this PR yet?

Reply:
The text editor that I use - Sublime - includes an option to delete all trailing whitespace every time you save, which has been very helpful. I'm not sure what editor you're using, Bryan, but you might have a similar option!

Reply:
You can have emacs take out trailing spaces.  See https://www.emacswiki.org/emacs/IndentingPython .